odo/backends/tests/test_dask.py

```diff
@@ -2,15 +2,19 @@
 
 import pytest
 pytest.importorskip('dask')
+pytest.importorskip('dask.bag')
```

Comment:
@mrocklin like this? I'm not sure what the build env would look like to cause that.